src/sources/mod.rs

```diff
@@ -131,7 +134,7 @@ pub enum Sources {
 
     /// EventStoreDB Metrics.
     #[cfg(feature = "sources-eventstoredb_metrics")]
-    EventstoreDbMetrics(#[configurable(derived)] eventstoredb_metrics::EventStoreDbConfig),
+    EventstoredbMetrics(#[configurable(derived)] eventstoredb_metrics::EventStoreDbConfig),
```

Comment:
Intentionally different capitalization between this and `eventstoredb_metrics::EventStoreDbConfig`?

Reply:
Yep, unfortunately.

Otherwise, the (de)serialized name becomes `eventstore_db_metrics` based on the capitilization. Changing it to this keeps its current name of `eventstoredb_metrics`.